### ISSUE TITLE
HELP!Use FCOS model train kitti(have been converted to VOC format),mAP is too bad,Emergency！

### ISSUE THREAD

**Issue by GYE19970220** [open-mmlab/mmdetection#4269]
**Describe**
I converted kitti to VOC format，train:val:test=7:2:1,the files under JPEGImages are copied from kitti image_2(xxxxxx.png).
I use fcos model with fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_dcn_4x4_1x_coco.py.
As follows, the result is very bad,even after running the 12th epoch.I changed lr=0.002, samples_per_gpu=16,workers_per_gpu=4,
But I don't know what went wrong，Do I need to increase the epoch？Or what‘s wrong with my data format? 
Any idea wold be appreciated.Thanks in advance!@ZwwWayne 
**Training message**
![image](https://user-images.githubusercontent.com/42958175/101632180-7c343600-3a60-11eb-87c2-1de7d64f243e.png)

**Conversion process**
First，handle the label of kitti,only save ‘Car’，‘Pedestrain’ and ‘Cyclist’，
![image](https://user-images.githubusercontent.com/42958175/101628955-8dc70f00-3a5b-11eb-9da6-aa0436063c4d.png)
![image](https://user-images.githubusercontent.com/42958175/101628985-9881a400-3a5b-11eb-83ba-9e3337695935.png)

The format convert file is：
![image](https://user-images.githubusercontent.com/42958175/101627904-02994980-3a5a-11eb-9961-e5ef57d82eec.png)
![image](https://user-images.githubusercontent.com/42958175/101629112-cc5cc980-3a5b-11eb-830d-d7728104eee2.png)
![image](https://user-images.githubusercontent.com/42958175/101629134-d5e63180-3a5b-11eb-880f-c31b5f44eb86.png)

After convert，the xml file example is：
![image](https://user-images.githubusercontent.com/42958175/101627673-a9311a80-3a59-11eb-8078-aa1d1434a0ff.png)
![image](https://user-images.githubusercontent.com/42958175/101627686-ae8e6500-3a59-11eb-85ce-7b2df9f5397f.png)

**Reproduction**
1.What command or script did you run?
`CUDA_VISIBLE_DEVICES=5 nohup python tools/train.py configs/fcos/fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_dcn_4x4_1x_coco.py --gpus=1 &`
2.Did you make any modifications on the code or config? Did you understand what you have modified?
I modified five files，they are respectively are 
①voc.py : classes
![image](https://user-images.githubusercontent.com/42958175/101629503-6ae92a80-3a5c-11eb-989e-5fa62c1c8c1f.png)

②voc0712.py : the data path
![image](https://user-images.githubusercontent.com/42958175/101629559-85230880-3a5c-11eb-8b5c-59116c958437.png)

③class_names.py : classes
![image](https://user-images.githubusercontent.com/42958175/101629588-910eca80-3a5c-11eb-9534-4d52012b53c2.png)

④fcos_r50_caffe_fpn_gn-head_4x4_1x_coco.py : _base_ and lr=0.002
![image](https://user-images.githubusercontent.com/42958175/101629776-cfa48500-3a5c-11eb-9d9f-9ebc2b3fc219.png)
![image](https://user-images.githubusercontent.com/42958175/101629809-e054fb00-3a5c-11eb-9283-e342e2fdc0a5.png)

⑤fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_dcn_4x4_1x_coco.py : samples_per_gpu=16
![image](https://user-images.githubusercontent.com/42958175/101629932-109c9980-3a5d-11eb-8346-9f1e1b1507f5.png)

**Environment**
sys.platform: linux
Python: 3.7.9 (default, Aug 31 2020, 12:42:55) [GCC 7.3.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: TITAN V
CUDA_HOME: /home/gye/cuda100
NVCC: Cuda compilation tools, release 10.0, V10.0.130
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.3.1
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.20.5 (Git Hash 0125f28c61c1f822fd48570b4c1066f96fcb9b2e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CUDA Runtime 10.0
  - NVCC architecture flags: -gencode;arch=compute_35,code=sm_35;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_50,code=compute_50
  - CuDNN 7.6.5
    - Built with CuDNN 7.6.4
  - Magma 2.5.0
  - Build settings: BLAS=MKL, BUILD_NAMEDTENSOR=OFF, BUILD_TYPE=Release, CXX_FLAGS=-fvisibility-inlines-hidden -std=c++11 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I/opt/conda/conda-bld/pytorch_1574381331675/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/include -fdebug-prefix-map=/opt/conda/conda-bld/pytorch_1574381331675/work=/usr/local/src/conda/pytorch-1.3.1 -fdebug-prefix-map=/opt/conda/conda-bld/pytorch_1574381331675/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place=/usr/local/src/conda-prefix -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -O2 -fPIC -Wno-narrowing -Wall -Wextra -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Wno-stringop-overflow, DISABLE_NUMA=1, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=True, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.4.2
OpenCV: 4.1.0
MMCV: 1.2.1
MMCV Compiler: GCC 7.5
MMCV CUDA Compiler: 10.0
MMDetection: 2.7.0+fb287b1

**Comment by ZwwWayne** [comment]
We have provided a [colab tutorial](https://github.com/open-mmlab/mmdetection/blob/master/demo/MMDet_Tutorial.ipynb) for converting KITTI to the coco format and finetune a detector on the dataset.
You may first check the tutorial and compare the implementation.
